ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -153,10 +155,17 @@ def install_mcg_only_cluster(self):
 
         """
         logger.info("Install MCG ONLY cluster via UI")
-        self.do_click(self.dep_loc["advanced_deployment"])
+        if self.ocp_version == "4.9":
+            self.do_click(self.dep_loc["advanced_deployment"])
         self.do_click(self.dep_loc["expand_advanced_mode"], enable_screenshot=True)
-        self.do_click(self.dep_loc["mcg_only_option"], enable_screenshot=True)
-        self.do_click(self.dep_loc["next"], enable_screenshot=True)
+        if self.ocp_version == "4.9":
+            self.do_click(self.dep_loc["mcg_only_option"], enable_screenshot=True)
+        elif self.ocp_version == "4.10":
+            self.do_click(self.dep_loc["mcg_only_option_4_10"], enable_screenshot=True)
+        if config.DEPLOYMENT.get("local_storage"):
+            self.install_lso_cluster()
+        else:
+            self.do_click(self.dep_loc["next"], enable_screenshot=True)
         self.do_click(self.dep_loc["next"], enable_screenshot=True)
         self.create_storage_cluster()
```

Comment:
For OCP version >= 4.9, it should be create storage system instead of storage cluster

Reply:
I added it on description..
https://github.com/red-hat-storage/ocs-ci/blob/deb3b32ab8378d01599a7873775e9e4319158b55/ocs_ci/ocs/ui/deployment_ui.py#L292